examples/shouldi/tests/test_dependency_check.py

```diff
@@ -0,0 +1,39 @@
+import pathlib
+
+from dffml.util.os import prepend_to_path
+from dffml.util.net import cached_download_unpack_archive
+from dffml.util.asynctestcase import AsyncTestCase
+
+from shouldi.dependency_check import run_dependency_check
+
+
+class TestRunDependencyCheckOp(AsyncTestCase):
+    @cached_download_unpack_archive(
+        "https://download.java.net/openjdk/jdk14/ri/openjdk-14+36_linux-x64_bin.tar.gz",
+        pathlib.Path(__file__).parent / "downloads" / "java.tar.gz",
+        pathlib.Path(__file__).parent / "downloads" / "java-download",
+        "d87ab7b623e17c85d763fd9bf810fc6de7d7c001facf238266bb316586081732cfd4b08d9fbaa83655cbdf9a4f497ac9",
+    )
+    @cached_download_unpack_archive(
+        "https://github.com/jeremylong/DependencyCheck/archive/v5.3.2.tar.gz",
```

Comment:
@pdxjohnny inside this package, there is no `bin` folder to point at. Please look if its right.
also, I looked and found a zip file which I converted in tar.gz -- `https://www.zamzar.com/download.php?uid=b6c3d067ebb13318f051adba339f863-e96eb4def3c783f&targetID=12Sr3NF0Ch82fNRomRbEjA_I_I&fileID=p1e5kdgueccns1fe11m7v1k0itqo4.tar.gz` inside that there is a `bin` folder. Please look out !!

Reply:
You're looking for the compiled version of DependencyCheck, which can be found at https://dl.bintray.com/jeremy-long/owasp/dependency-check-5.3.2-release.zip